CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Add `iconPosition` property to `Input` component @mnajdova ([#442](https://github.com/stardust-ui/react/pull/442))
 - Add `color`, `inverted` and `renderContent` props and `content` slot to `Segment` component @Bugaa92 ([#389](https://github.com/stardust-ui/react/pull/389))
 - Add focus trap behavior to `Popup` @kuzhelov ([#457](https://github.com/stardust-ui/react/pull/457))
+- Change `createShorthandFactory` API, added `as` prop functionality in order to deprecate HTML factories @Bugaa92 ([#376](https://github.com/stardust-ui/react/pull/376))
```

Comment:
Since this change is not public facing, we don't need to add a changelog entry.

Reply:
removed, thanks